nrf_modem/doc/dectphy.rst

```diff
@@ -72,7 +72,16 @@ Before initializing the DECT PHY interface of the Modem library, the application
 Afterwards, the application can initialize the DECT PHY interface by calling the :c:func:`nrf_modem_dect_phy_init` function.
 Upon successful initialization, both the DECT PHY interface and DECT NR+ physical layer in the modem are ready for operation.
 
-The very first time the DECT PHY interface is initialized on a nRF9131 SiP, the DECT NR+ PHY firmware locks the calibration data preventing any changes to it.
+On nRF9151 devices only, the application can configure band 4 support during initialization by setting the :c:member:`nrf_modem_dect_phy_init_params.band4_support` field to 1.
+
+.. note::
+   DECT NR+ band 4 is supported by the nRF9151 LACA A0 SiP for R&D evaluation purposes.
+   The nRF9151 LACA A0AB SiP will be qualified for DECT NR+ band 4 support.
+
+.. important::
+   When operating on band 4, the carries outside of the [525, 551] range shall not be used as they interfere with other radio devices, including LTE devices.
+
+On nRF9131 devices only, the DECT NR+ PHY firmware locks the calibration data the very first time the DECT PHY interface is initialized.
```

Comment:
```suggestion
On nRF9131 Series devices only, the DECT NR+ PHY firmware locks the calibration data the first time the DECT PHY interface is initialized.
```

Reply:
I have added `Series` but kept `very` to emphasize the fact that this operation happens only once on the device (during its entire lifetime). The interface itself can be initialized multiple times at runtime (e.g. `_init()`, `_deinit()`, `_init()`) and during the device liftime, but only the very first time that `_init()` it is called on the device something is happening.